src/components/FloatingButton.tsx

```diff
@@ -0,0 +1,50 @@
+import { FloatingAction } from "react-native-floating-action";
+import {View, Text, StyleSheet} from 'react-native';
+import React from 'react';
+  
+
+const actions = [
+    {
+      text: "Assigned Goal",
+      icon: require('../../assets/assign-user-solid.237x256.png'),
+      buttonSize:50,
+      name: "bt_assignedGoal",
+      position: 1,
+      color : '#492ed1' 
+    },
+    {
+      text: "New Goal",
+      icon: require("../../assets/target.256x256.png"),
+      buttonSize:50,
+      name: "bt_newGoal",
+      position: 2,
+      color : '#492ed1' 
+    },
+    {
+      text: "To Do",
+      icon: require("../../assets/task.201x256.png"),
+      buttonSize:50,
+      name: "bt_todo",
+      position: 3,
+      color : '#492ed1' 
+    },
+  ];
```

Comment:
Can we move the actions in a different file to make it a bit cleaner

Reply:
Sure

Reply:
Done